src/tilemaps/components/StaggeredTileToWorldY.js

```diff
@@ -32,7 +32,7 @@ var StaggeredTileToWorldY = function (tileY, camera, layer)
         tileHeight *= tilemapLayer.scaleY;
     }
 
-    return layerWorldY + tileY * (tileHeight / 2);
+    return layerWorldY + tileY * (tileHeight / 2) + tileHeight;
```

Comment:
I'm not entirely sure why this was necessary. Without the change, all colliders were set half a tile height above where they should have been.

With the change (note how the rectangle encompasses the correct tile for the tall grass and the custom collider is at the bottom of the wall tiles)
![image](https://user-images.githubusercontent.com/6614995/123968728-ca3a6800-d9b7-11eb-8fe1-9c38bf8290bf.png)

Without the change
![image](https://user-images.githubusercontent.com/6614995/123969010-1685a800-d9b8-11eb-8ce0-a8a551773d4e.png)